### PR TITLE
adds nvmrc for v18

### PR DESCRIPTION
**Pull request by peterpme**
.nvmrc works with fnm and nvm

### CODE REVIEW

**Comment by wentokay** [comment]
it might be worth adding something like this in the root package.json too for people who aren't using a version manager

```
{
  "engines": {
    "node": "18.x"
  }
}
```

**Comment by wentokay** [comment]
maybe a yarn major version in engines too, I'm using 1.22.19 atm. (upgrading the major version or moving to pnpm one day could be nice)

it might not be necessary for the engines stuff though, wdyt @peterpme?

**Comment by peterpme** [comment]
Great idea!

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -275,7 +275,7 @@ jobs:
       # NPM Publish.
       ##########################################################################
 
-      - name:  "update npm package version"
+      - name: "update npm package version"
```
Comment (peterpme) [inline .github/workflows/pull_request.yml:278]: random space heh

**Pull request merged**